biz.aQute.tester.junit-platform/src/aQute/tester/junit/platform/Activator.java

```diff
@@ -210,6 +237,24 @@ public void run() {
 		}
 	}
 
+	String testName(TestIdentifier testIdentifier) {
+		return testIdentifier.getSource()
+			.map(source -> {
+				if (source instanceof ClassSource) {
+					ClassSource classSource = (ClassSource) source;
+
+					return classSource.getClassName();
+				} else if (source instanceof MethodSource) {
+					MethodSource methodSource = (MethodSource) source;
+
+					return methodSource.getClassName() + ":" + methodSource.getMethodName();
```

Comment:
MethodSource can also include parameters (`MethodSource.getMethodParameterTypes()`). So if the value of that is not null, you should add it to the string. Also, in JUnit 5, they use `#` to separate the class name from the method name. Bnd will accept either in the specified test name. See 

https://github.com/bndtools/bnd/blob/17f24db7aea5a37fa38dd691483ac15c99158de1/biz.aQute.tester.junit-platform/src/aQute/tester/junit/platform/utils/BundleUtils.java#L91

Reply:
How should we represent parameters? appended surrounded by parens?

Reply:
check updated.